autorest/codegen/templates/templates_package/setup.py.jinja2

```diff
@@ -1,10 +1,6 @@
-#!/usr/bin/env python
-
-# -------------------------------------------------------------------------
-# Copyright (c) Microsoft Corporation. All rights reserved.
-# Licensed under the MIT License. See License.txt in the project root for
-# license information.
-# --------------------------------------------------------------------------
+# coding=utf-8
```

Comment:
@msyyc it would be great if this could be combined with the setup.py existing in `autorest/codegen/templates/setup.py.jinja2`. They're really quite similar, this one just has extra code. I think that also means not making a separate folder for templates_package